src/sentry/sentry_metrics/utils.py

```diff
@@ -27,6 +28,48 @@ def reverse_resolve_tag_value(
             return reverse_resolve(use_case_id, org_id, index)
 
 
+def bulk_reverse_resolve_tag_value(
+    use_case_id: UseCaseID, org_id: int, indexes: Collection[Union[int, str, None]]
```

Comment:
nit: as the comment below explains these are not really `indexes`. Maybe call it `elements`, `elements_to_resolve` or `values_to_resolve`? 🤷

Reply:
thanks for the input,
I'll call them `values` it's a bit too generic, but `indexes` is not quite correct.